hummingbot/market/coinbase_pro/coinbase_pro_market.pyx

```diff
@@ -800,39 +801,33 @@ cdef class CoinbaseProMarket(MarketBase):
                 app_warning_msg=f"Failed to cancel the order {order_id} on Coinbase Pro. "
                                 f"Check API key and network connection."
             )
-        return order_id
+        return None
 
     cdef c_cancel(self, str symbol, str order_id):
         asyncio.ensure_future(self.execute_cancel(symbol, order_id))
         return order_id
 
     async def cancel_all(self, timeout_seconds: float) -> List[CancellationResult]:
         incomplete_orders = [o for o in self._in_flight_orders.values() if not o.is_done]
-        exchange_order_id_map = dict(zip([o.exchange_order_id for o in incomplete_orders], incomplete_orders))
+        tasks = [self.execute_cancel(o.symbol, o.client_order_id) for o in incomplete_orders]
+        order_id_set = set([o.client_order_id for o in incomplete_orders])
         successful_cancellations = []
 
         try:
-            path_url = "/orders"
             async with timeout(timeout_seconds):
-                results = await self._api_request("delete", path_url=path_url)
-
-            for exchange_order_id in results:
-                order = exchange_order_id_map.get(exchange_order_id)
-                if order is not None:
-                    exchange_order_id_map.pop(exchange_order_id, None)
-                    successful_cancellations.append(CancellationResult(order.client_order_id, True))
-        except asyncio.CancelledError:
-            raise
+                results = await asyncio.gather(*tasks, return_exceptions=True)
+                for client_order_id in results:
+                    if type(client_order_id) is str:
+                        order_id_set.remove(client_order_id)
+                        successful_cancellations.append(CancellationResult(client_order_id, True))
         except Exception:
             self.logger().network(
                 f"Unexpected error cancelling orders.",
                 exc_info=True,
-                app_warning_msg=f"Failed to cancel orders on Coinbase Pro. "
-                                f"Check API key and network connection."
+                app_warning_msg="Failed to cancel order with Binance. Check API key and network connection."
```

Comment:
nit: "Failed to cancel order*s* with Binance ..."

Reply:
fixed